src/main/scala/com/lambtors/poker_api/module/poker/application/player_cards/find/PlayerCardsFinder.scala

```diff
@@ -10,6 +10,7 @@ final class PlayerCardsFinder[P[_]: MonadErrorThrowable](playerRepository: Playe
   def find(playerId: PlayerId): P[(Card, Card)] =
     playerRepository
       .search(playerId)
-      .flatMap(_.fold[P[(Card, Card)]](MonadErrorThrowable[P].raiseError(PlayerNotFound(playerId)))(player =>
-        (player.firstCard, player.secondCard).pure[P]))
+      .fold[P[(Card, Card)]](MonadErrorThrowable[P].raiseError(PlayerNotFound(playerId)))(player =>
+        (player.firstCard, player.secondCard).pure[P])
```

Comment:
(player.firstCard, player.secondCard) => player.cards (method that returns a tuple inside the case class)?